resnet50/resnet50_benchmark.xml

```diff
@@ -286,22 +285,29 @@
+      <do  mode="shell" tag="(H100|A100|WAIH100)+synthetic">printf "%s\n" "export PYTHONPATH=${packages}:\$PYTHONPATH" "\$*" > ${jube_benchmark_home}/${jube_wp_relpath}/nvidia_x86_tensorflow_wrap.sh</do>
+      <do  mode="shell" tag="(GH200|JEDI)+synthetic">printf "%s\n" "export PYTHONPATH=${packages}:\$PYTHONPATH" "\$*" > ${jube_benchmark_home}/${jube_wp_relpath}/nvidia_arm_tensorflow_wrap.sh</do>
+      <do  mode="shell" tag="MI250+synthetic">printf "%s\n" "export PYTHONPATH=${packages}:\$PYTHONPATH" "\$*" > ${jube_benchmark_home}/${jube_wp_relpath}/amd_tensorflow_wrap.sh</do>
+      <do  mode="shell" tag="GC200+!synthetic">printf "%s\n" "export PYTHONPATH=${packages}:\$PYTHONPATH" "${data_path}" "\$*" > ${jube_benchmark_home}/${jube_wp_relpath}/ipu_tensorflow_wrap.sh</do>
+      <do  mode="shell" tag="(H100|A100|WAIH100)+!synthetic">printf "%s\n" "export PYTHONPATH=${packages}:\$PYTHONPATH" "${data_path}" "\$*" > ${jube_benchmark_home}/${jube_wp_relpath}/nvidia_x86_tensorflow_wrap.sh</do>
+      <do  mode="shell" tag="(GH200|JEDI)+!synthetic">printf "%s\n" "export PYTHONPATH=${packages}:\$PYTHONPATH" "${data_path}" "\$*" > ${jube_benchmark_home}/${jube_wp_relpath}/nvidia_arm_tensorflow_wrap.sh</do>
+      <do  mode="shell" tag="MI250+!synthetic">printf "%s\n" "export PYTHONPATH=${packages}:\$PYTHONPATH" "${data_path}" "\$*" > ${jube_benchmark_home}/${jube_wp_relpath}/amd_tensorflow_wrap.sh</do>
+      <do  mode="shell" tag="GC200"> chmod a+rwx ${jube_benchmark_home}/${jube_wp_relpath}/ipu_tensorflow_wrap.sh </do>
+      <do  mode="shell" tag="(H100|A100|WAIH100)"> chmod a+rwx ${jube_benchmark_home}/${jube_wp_relpath}/nvidia_x86_tensorflow_wrap.sh </do>
+      <do  mode="shell" tag="(GH200|JEDI)"> chmod a+rwx ${jube_benchmark_home}/${jube_wp_relpath}/nvidia_arm_tensorflow_wrap.sh </do>
+      <do  mode="shell" tag="MI250"> chmod a+rwx ${jube_benchmark_home}/${jube_wp_relpath}/amd_tensorflow_wrap.sh </do>
       <do>$submit $submit_script</do>
       <do done_file="$done_file"></do>
     </step>
 
-   <step name="execute" tag="!container+!GC200" iterations="1">
+   <step name="combine_energy" tag="!container" depend="execute" iterations="1">
       <use>files</use>
       <use>systemInfo</use>
       <use>systemParameter</use>
-      <use>modelParameter</use>
-      <use>environment</use>
-      <use>executeset</use>
-      <use from="platform.xml">jobfiles</use>
-      <use from="platform.xml">executesub</use>
-      <do>$submit $submit_script</do>
-      <do done_file="$done_file"></do>
-    </step>
+      <do>module load GCC SciPy-bundle</do>
+      <do>python3 ${jube_benchmark_home}/aux/combine_energy.py --output ${jube_benchmark_home}/${jube_wp_relpath}/combined_energy.csv $$(printf "${jube_benchmark_home}/${jube_wp_relpath}/execute/${system_name}_power.%d.energy.csv " $$(seq 0 $$(($nodes-1))))</do>
```

Comment:
Ah, `$nodes` should be replaced with number of ranks instead, same for llm actually

Reply:
in this expression: `$$(seq 0 $$(($nodes-1))))`